src/aiperf/dataset/agentic_code_gen/reporting/trace.py

```
@@ -0,0 +1,148 @@
+    errors: list[str] = []
+    line_count = 0
+
+    with path.open("rb") as f:
+        for line_num, line in enumerate(f, 1):
+            line = line.strip()
+            if not line:
+                continue
+            line_count += 1
+            try:
+                MooncakeTrace(**orjson.loads(line))
+            except Exception as e:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)